nixos/modules/services/mail/postfix.nix

```diff
@@ -584,6 +516,163 @@ in
               ])
             );
           options = {
+            message_size_limit = mkOption {
+              type = with types; nullOr int;
+              default = null;
+              example = "52428800"; # 50 MiB
+              description = ''
+                Maximum size of an email message in bytes.
+
+                <https://www.postfix.org/postconf.5.html#message_size_limit>
+              '';
+            };
+
+            mydestination = mkOption {
+              type =
+                with types;
+                nullOr (oneOf [
+                  str
+                  (listOf str)
+                ]);
+              default = null;
+              description = ''
+                List of domain names intended for local delivery using /etc/passwd and /etc/aliases.
+
+                ::: {.warning}
+                Do not include [virtual](https://www.postfix.org/VIRTUAL_README.html) domains in this list.
+                :::
+
+                Leave unset for Postfix to populate this with reasonable defaults.
+
+                <https://www.postfix.org/postconf.5.html#mydestination>
+              '';
+            };
+
+            myhostname = mkOption {
+              type = with types; nullOr types.str;
+              default = null;
+              example = "mail.example.com";
+              description = ''
+                The fully-qualified domain name of the system.
+
+                Leave unset for Postfix to populate this value with the local hostname.
```

Comment:
That's not quite correct.

Reply:
Would moving to hostname be more correct?

```
❯ postconf -d | grep "^myhostname"

postconf: warning: open /etc/postfix/master.cf: No such file or directory
myhostname = gaia.localdomain
```

> The internet hostname of this mail system. The default is to use the fully-qualified domain name (FQDN) from gethostname(), or to use the non-FQDN result from gethostname() and append ".$[mydomain](https://www.postfix.org/postconf.5.html#mydomain)". $[myhostname](https://www.postfix.org/postconf.5.html#myhostname) is used as a default value for many other configuration parameters.

https://www.postfix.org/postconf.5.html#myhostname